{"uuid": "6fe527c9-1f47-4246-a00b-57cc6ec011d1", "children": ["30596632-0c0e-41cf-95c3-dbf181bc590b"], "befores": [{"name": "test_logintopage", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status\nfrom disconnected: received Inspector.detached event\n  (Session info: chrome=102.0.5005.63)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x005ED953+2414931]\n\tOrdinal0 [0x0057F5E1+1963489]\n\tOrdinal0 [0x0046C6B8+837304]\n\tOrdinal0 [0x0045E45E+779358]\n\tOrdinal0 [0x0045E04F+778319]\n\tOrdinal0 [0x0045D646+775750]\n\tOrdinal0 [0x0045C565+771429]\n\tOrdinal0 [0x0045CB68+772968]\n\tOrdinal0 [0x00468B10+822032]\n\tOrdinal0 [0x0045DE5D+777821]\n\tOrdinal0 [0x0045EA3B+780859]\n\tOrdinal0 [0x0045E06A+778346]\n\tOrdinal0 [0x0045D646+775750]\n\tOrdinal0 [0x0045C565+771429]\n\tOrdinal0 [0x0045CA3D+772669]\n\tOrdinal0 [0x0046DEA4+843428]\n\tOrdinal0 [0x004C4EBD+1199805]\n\tOrdinal0 [0x004B449C+1131676]\n\tOrdinal0 [0x004C4812+1198098]\n\tOrdinal0 [0x004B42B6+1131190]\n\tOrdinal0 [0x0048E860+976992]\n\tOrdinal0 [0x0048F756+980822]\n\tGetHandleVerifier [0x0085CC62+2510274]\n\tGetHandleVerifier [0x0084F760+2455744]\n\tGetHandleVerifier [0x0067EABA+551962]\n\tGetHandleVerifier [0x0067D916+547446]\n\tOrdinal0 [0x00585F3B+1990459]\n\tOrdinal0 [0x0058A898+2009240]\n\tOrdinal0 [0x0058A985+2009477]\n\tOrdinal0 [0x00593AD1+2046673]\n\tBaseThreadInitThunk [0x750FFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77167A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77167A4E+238]\n\n", "trace": "  File \"c:\\users\\user\\pycharmprojects\\orangehrm\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\user\\pycharmprojects\\orangehrm\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\user\\pycharmprojects\\orangehrm\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\User\\PycharmProjects\\orangehrm\\testcases\\test_login.py\", line 20, in test_logintopage\n    self.driver.get(self.URL)\n  File \"c:\\users\\user\\pycharmprojects\\orangehrm\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 442, in get\n    self.execute(Command.GET, {'url': url})\n  File \"c:\\users\\user\\pycharmprojects\\orangehrm\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 430, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\user\\pycharmprojects\\orangehrm\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1654840942036, "stop": 1654840943130}], "start": 1654840942036, "stop": 1654840943500}